db/seeds/support/data_migrator.rb

```diff
@@ -118,6 +158,29 @@ def person_from_row(row)
 
     private
 
+    def dachverband_mailing_list(key)
+      name = {
+        advertising: 'Werbung SVSE',
+        newsletter_dachverband_subscriber: 'Newsletter',
+        bulletin_subscriber: 'Bulletin'
+      }[key]
+
+      MailingList.find_or_create_by(name: name,
+                                    group_id: Group.find_by(name: 'SVSE').id,
+                                    subscribable: true)
+    end
+
+    def section_mailing_list(key, group)
+      name = {
+        versand_subscriber: 'Versand',
+        newsletter_section_subscriber: 'Newsletter',
+      }[key]
+
+      MailingList.find_or_create_by(name: name,
+                                    group_id: group.layer_group.id,
+                                    subscribable: true)
+    end
```

Comment:
Ich habe das Gefühl, dass man das besser machen kann. Hierfür ist es erstmal gut, ich versuche den Usecase (Foreignkeys) dann auch in meiner Doku für den nächsten Datentransfer festzuhalten. Sowas haben wir ja sicher öfter...